src/Promitor.Integrations.Sinks.Prometheus/Collectors/AzureScrapingSystemMetricsPublisher.cs

```diff
@@ -48,5 +48,14 @@ public async Task WriteGaugeMeasurementAsync(string name, string description, do
         {
             await _systemMetricsPublisher.WriteGaugeMeasurementAsync(name, description, value, labels, includeTimestamp);
         }
+        
+        public async Task WriteHistogramMeasurementAsync(string name, string description, double value, Dictionary<string, string> labels)
+        {
+            throw new System.NotImplementedException();
```

Comment:
Let's not throw but log or ignore instead; this would break end-users using this sink